sign_oca/models/sign_oca_request.py

```diff
@@ -73,7 +73,7 @@ class SignOcaRequest(models.Model):
     def _compute_next_item_id(self):
         for record in self:
             record.next_item_id = (
-                record.signatory_data and max(record.signatory_data.keys()) or 0
+                record.signatory_data and int(max(record.signatory_data.keys())) or 0
```

Comment:
Reviewing this is probably a bad decision, we should need to do:
`max([int(key) for key in record.signatory_data.keys()])`

For example, if you do max(['9', '10']) the result is '9'